Project/Project/Services/AiApiService.cs

```diff
@@ -0,0 +1,64 @@
+using System.Text;
+
+using System.Text.Json;
+
+namespace Project.Services;
+
+public class AiApiService
+{
+    private static readonly HttpClient client = new HttpClient();
+    
+    private static readonly IConfigurationRoot Myconfig = new ConfigurationBuilder().AddJsonFile("appsettings.json").Build();
+    private static readonly string? ApiURL = Myconfig.GetValue<string>("Api:AiURL");
+    
+    public static async Task<string> GenerateResponse(string prompt)
+    {
+        var requestBody = new
+        {
+            model = "movie-critic:latest",
+            prompt = prompt,
+            stream = false
+        };
+
+        var json = JsonSerializer.Serialize(requestBody);
+        var content = new StringContent(json, Encoding.UTF8, "application/json");
+
+        var response = await client.PostAsync(ApiURL, content);
+
+        if (response.IsSuccessStatusCode)
+        {
+            var responseContent = await response.Content.ReadAsStringAsync();
+            var responseObject = JsonSerializer.Deserialize<JsonElement>(responseContent);
+
+            if (responseObject.TryGetProperty("response", out var responseProperty))
+            {
+                var responseString = responseProperty.GetString();
+                var jsonContent = ExtractJsonFromMarkdown(responseString);
```

Comment:
I would:
Removed static keywords.
Injected HttpClient and IConfiguration through the constructor. What do you think